blocklylib-core/src/main/java/com/google/blockly/android/ui/AbstractBlockView.java

```diff
@@ -370,7 +373,30 @@ protected boolean isEntireBlockHighlighted() {
      * @return True if the coordinate of the motion event is on the visible, non-transparent part of
      * this view; false otherwise.
      */
-    protected abstract boolean hitTest(MotionEvent event);
+    protected boolean hitTest(MotionEvent event) {
```

Comment:
In the JavaDoc above I should have noted whether we are assuming this event is for this View.  I think that is true, but it would be better to make that explicit.